config-templates/netapp/netapp-ontap-san/20.07/custom-parameters.json

```
@@ -0,0 +1,55 @@
+[
+  {
+    "description": "Namespace for driver deployment",
```

If they specify a namespace here, do their apps also need to be in this namespace?

```suggestion
    "description": "The namespace to install the storage drivers.",
```